src/core/DataFile.cpp

```diff
@@ -883,6 +883,21 @@ void DataFile::upgrade_1_1_91()
 			el.setAttribute( a, ( el.attribute( a ) == "0" ) ? "1" : "0" );
 		}
 	}
+
+	list = elementsByTagName( "arpeggiator" );
+	for( int i = 0; !list.item( i ).isNull(); ++i )
+	{
+		QDomElement el = list.item( i ).toElement();
+		// Swap elements ArpDirRandom and ArpDirDownAndUp
+		if( el.attribute( "arpdir" ).toInt() == 3 )
```

Comment:
I tried that first and it wouldn't compile. I had another go and it came out like this instead. Looks cleaner so I'm changing.

```
		if( el.attribute( "arpdir" ) == "3" )
		{
			el.setAttribute( "arpdir", "4" );
		}
```